README.md

```diff
@@ -0,0 +1,112 @@
+# :green_book: Table of Contents
+
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Project-Presentation](#project-presentation)
+  - [Live-Demo](#live-demo)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# :book: [Portfolio]
+
+**[Capstone Project]** is a microverse project where I created a home and about page using HTML, CSS & Javascript. In this project I included 2 pages i.e. home page and about page.
+
+## :hammer_and_wrench: Built With
+
+### Tech Stack
+
+**Client**
+HTML, CSS and Javascript
+
+### Key Features
+
+- **[- Use Javascript, HTML & CSS linters]**
+- **[- Add Mobile Menu Toggle using Javascript]**
+- **[- The section "Featured speakers" in the home page created dynamically using JavaScript.]**
+- **[- Created Responsiveness for both mobile and desktop version.]**
+- **[- Use Flexbox & Grid]**
+
+## :computer: Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a web browser, node, npm and an IDE like VS Code should be installed in your computer.
+
+### Setup
+
+Open Terminal & Clone this repository to your desired folder:
+
+```sh
+cd <path-to-the-folder>
+git clone https://github.com/tasnimchy96/portfolio.github.io
+cd portfolio.github.io
+code .
+```
+### Install
+
+Install this project using the cloned repo or download and use the files to your local machine.
+
+```sh
+cd <path-to-the-cloned-repo>
+code .
+```
+### Usage
+
+To run the project open the index.html file in your web browser or you can also open live server from the IDE.
+
+### Run tests
+
+No test are available
+
+### Project-Presentation
+-[Project-Presentaion](https://www.loom.com/share/1a1fba5b911d4a8c85e733ff2da3d25c)
+
+### :rocket: Live-Demo
+
+- [Live-Demo](https://tasnimchy96.github.io/portfolio.github.io/)
```

Comment:
- [ ] Please kindly include the live demo of your capstone project as this is that of your portfolio.